src/potions.ts

```diff
@@ -434,3 +430,121 @@ export function bathroomFinance(embezzlers: number): void {
     }
   }
 }
+
+class VariableMeatPotions {
```

Comment:
vibe-wise I expect this to extend Potion--I am incredibly unmarried to this. I cannot imagine being less married to something. Also, should be called `VariablePotion` singular, probably